Codeinwp/neve
# Feat: Allow custom SVG input on icon selectors

### Summary

Adds an option for custom SVG input in the following icon selectors: palette switch, cart icon, and my account icon.
This is a PRO feature, related to https://github.com/Codeinwp/neve-pro-addon/pull/1822.

### Will affect visual aspect of the product

NO

### Screenshots 
![image](https://user-images.githubusercontent.com/39873395/149951519-79c6b0dc-0834-4205-8ca0-ad93d2771c04.png)

### Test instructions


- Download together with the PR mentioned above
- Go to the icon selectors from the palette switch, cart icons, and my account icon
- Next to the existing icons should be an `editor-code` one that slides down a text area when clicked
- Enter an SVG code and click Save for the changes to apply
- The new icon should display fine and the dimensions should correspond to the value from the Icon Size setting


Closes [Codeinwp/neve-pro-addon#1648](https://github.com/Codeinwp/neve-pro-addon/issues/1648).
